test/e2e/features/step_definitions/timestamp.steps.js

```diff
@@ -0,0 +1,21 @@
+var CaseManager = require('../pageObjects/common/CaseManager');
+
+var { defineSupportCode } = require('cucumber');
+
+defineSupportCode(function ({ And, But, Given, Then, When }) {
+
+  let caseManager = new CaseManager();
+
+    When('I Add Comment to the case', async function () {
+      await caseManager.startNextStep('Add Comment');
+      await caseManager.submitComment();
+    });
+
+
+    Then('I see the event with the correct timestamp', async function () {
```

Comment:
here we validating event time stamp to be current time, can you update step to reflect same "I see the event with current timestamp"

Reply:
done